src/environment/src/env/GazeboEnv.py

```diff
@@ -143,8 +169,13 @@ def _load_board(self):
         image_array = np.array(board_image) * 255
         return image_array
 
-    def _print_car_position_on_board(self, car_x, car_y):
+    def _print_car_position_on_board(self, car_x, car_y, closest_point):
+        pnt_r, pnt_c = closest_point
         distance = 15
-        img = self.board
+        img = self.board.copy()
         img[car_x - distance:car_x + distance, car_y - distance:car_y + distance] = 255
-        toimage(img).show()
+        img[pnt_r - distance:pnt_r + distance, pnt_c - distance:pnt_c + distance] = 255
+        img = toimage(img)
+        # img.show() # I was not able to make it work on my machine (in docker). Image is saved instead.
+        img.save("/home/ghost/ghost-racer/src/environment/src/env/data/car_and_closest_point.jpeg")
```

Comment:
This path should not be hardcoded, instead you could use `os.getcwd()` or relative path ;)

Reply:
I chagned this `save`, so the path is not hardcoded anymore. Actual path is stored in `self.cwd`.